tests/test_example.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio Resources module to create REST APIs"""
+
+
+def test_create_record(base_client):
+    """Test record creation."""
+
+    # Create draft
+    draft = base_client.post("/records")
+    assert draft.status_code == 201  # Draft created
+
+    # Get draft id
+    rec_uuid = draft.json.get("id", None)
+    assert rec_uuid is not None
+
+    # Update with draft content
+    draft_content = draft.json.get("metadata")
+
+    # TODO: Add some new metadata
+    draft = base_client.put(
+        "/records/{}/draft".format(rec_uuid), data=draft_content
+    )
+    assert draft.status_code == 200
+
+    # TODO: Add case where validation fails
```

Comment:
True, validation should probably be a different test module since it involves many things:
- malformed
- permissions
- content schema
- other content validations
- probably more